.github/workflows/garm-docker-image.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build and push garm docker image
+on: workflow_dispatch
+
+jobs:
+  Build-and-Push-Image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NAME: confidential-containers/garm
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this automatically populated?

Reply:
I ask this because for ACR we don't need secrets to push to it and pull from.

Reply:
I guess for GHCR this will be needed since we will be automatically building the image on each merge or whatever.

Reply:
this is an automatically created token available for workflows, I suppose it can push image, clone repos, etc.